manifests/init.pp

```diff
@@ -129,6 +130,9 @@
   }
 
   # validate params
+  validate_integer($vas_conf_vasypd_update_interval)
+  $vas_conf_vasypd_update_interval_integer = 0 + $vas_conf_vasypd_update_interval # str2integer
```

Comment:
When done, please remove the gem related changes and squash the functional commits into one so that I can merge them fast :)